tiny_dnn/layers/recurrent_cell_layer.h

```diff
@@ -65,9 +65,22 @@ class recurrent_cell_layer : public layer {
                        bool has_bias                = true,
                        activation_layer *activation = new tanh_layer,
                        core::backend_t backend_type = core::default_engine())
-    : layer(recurrent_order(has_bias),  // output bias
-            {vector_type::data,         // output vector
-             vector_type::aux}) {
+    : layer({vector_type::data, vector_type::data, vector_type::data,
+             vector_type::data, vector_type::data},
+            {vector_type::data, vector_type::data}) {
```

Comment:
apparently in the new api all this arguments has to be data type. @karandesai-96